src/index.js

```diff
@@ -105,7 +105,7 @@ class LinesEllipsis extends React.PureComponent {
       clamped,
       text: clamped ? this.units.slice(0, ellipsisIndex).join('') : props.text
     }
-    this.setState(newState, props.onReflow.bind(this, newState))
+    this.setState(newState, props.onReflow && props.onReflow.bind(this, newState))
```

Comment:
`defaultProps.onReflow` has been set as a no-op function, so I think the `&&` test is not necessary.

Reply:
Nice catch, I added that early on and forgot to remove it. Let me know if there is anything else you would like me to change.